octobot_commons/databases/adaptors/abstract_database_adaptor.py

```diff
@@ -14,6 +14,8 @@
 #
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
+import octobot_commons.multiprocessing_util as multiprocessing_util
+import octobot_trading.enums as trading_enums
```

Comment:
I don't think we should import `octobot_trading` here

Reply:
Indeed !